cypress/helpers/constants.js

```diff
@@ -0,0 +1,7 @@
+export const DEFAULT_CONFIG = {
+  defaultCommandTimeout: 15000,
+  viewportWidth: 2000,
+  viewportHeight: 3500,
+}
```

Comment:
is there a way to configure this part globally?

Reply:
Yes I think so! will take a look